application/paths.py

```
@@ -18,13 +18,18 @@
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
 
+import logging
 import os
 import platform
 
 
+logger = logging.getLogger(__name__)
+
+
 def makedir_if_not_exists(path):
```

Ruff

Missing return type annotation for public function `makedir_if_not_exists`

Add return type annotation: `None`

(ANN201)